src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -43,6 +53,30 @@ public void setOutput(double output) {
     mRoller.set(ControlMode.PercentOutput, output);
   }
 
+  public boolean isBallDetected(){
```

Comment:
I cannot figure out what this method is trying to do with all of these counters. Max says that it's code that was already there, but it's not obvious. Comments would be great.

Reply:
This was clarified with new comments